doc/formats.md

```diff
@@ -745,6 +746,17 @@ fq '.tcp_connections[] | select(.server.port=="rtmp") | d' file.cap
 - https://rtmp.veriskope.com/docs/spec/
 - https://rtmp.veriskope.com/pdf/video_file_format_spec_v10.pdf
 
+## tzif
+
+### Authors
+- Takashi Oguma
+[@bitbears-dev](https://github.com/bitbears-dev)
+[@0xb17bea125](https://twitter.com/0xb17bea125)
+
+### References
+- https://datatracker.ietf.org/doc/html/rfc8536
+
+
```

Comment:
Move to `tzif.md`, then this will be generated by `make doc` and also `fq -h tzif` will work.

Reply:
done @ [09c1f7e](https://github.com/wader/fq/pull/498/commits/09c1f7e99977d38519f9b9bb9c25fb5efcf01471)